contrib/pg_get_functiondef/meson.build

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) 2024, PostgreSQL Global Development Group
+
+# .. so that includes of pg_get_functiondef work
+pg_get_functiondef_inc = include_directories('.', '../')
```

Comment:
I know you copied the meson.build of hstore, but I believe that you don't need pg_get_functiondef_inc as you have no header file at all. 

BTW, IvorySQL doesn't support meson build yet, have you tried this kind of build on IvorySQL?